servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/LoadBalancerBuilder.java

```diff
@@ -124,56 +114,6 @@ LoadBalancerBuilder<ResolvedAddress, C> outlierDetectorFactory(
      */
     LoadBalancerBuilder<ResolvedAddress, C> linearSearchSpace(int linearSearchSpace);
 
-    // TODO: these healthCheck* methods should be moved into their own OutlierDetection configuration instance
-    //  and much like the LoadBalancingPolicy, we should be able to add `OutlierDetectionPolicy`s
-    /**
-     * Configure an interval for health checking a host that failed to open connections. If no interval is provided
-     * using this method, a default value will be used.
-     * <p>
-     * {@link #healthCheckFailedConnectionsThreshold(int)} can be used to disable the health checking mechanism
-     * and always consider all hosts for establishing new connections.
-     *
-     * @param interval interval at which a background health check will be scheduled.
-     * @param jitter the amount of jitter to apply to each retry {@code interval}.
-     * @return {@code this}.
-     * @see #healthCheckFailedConnectionsThreshold(int)
-     */
-    LoadBalancerBuilder<ResolvedAddress, C> healthCheckInterval(Duration interval, Duration jitter);
```

Comment:
The removal of these methods is part of the biggest change in the PR: modification of the API by moving the outlier detector configuration all to `OutlierDetectorConfig`.